src/utils/fusionAuth.ts

```diff
@@ -0,0 +1,150 @@
+import fs from 'fs-extra';
+import os from 'os';
+import path from 'path';
+import prompts from 'prompts';
+import Webhead, { WebheadInstance } from 'webhead';
+
+const builderApiURL = '{HOST}/api/builder';
+const fusionAuthURL = 'https://fusionauth-ce.betty.services';
+
+type response = string | object | null;
+
+class FusionAuth {
+  private configFile: string;
+
+  private host: string;
+
+  public loginId?: string;
+
+  public password?: string;
+
+  private relogin: () => Promise<void>;
+
+  private webhead: WebheadInstance;
+
+  constructor(host: string, relogin: () => Promise<void>) {
+    this.configFile = path.join(os.homedir(), '.bb-cli-fa');
+    this.host = host;
+    this.relogin = relogin;
+    this.webhead = Webhead();
+  }
+
+  async ensureLogin(): Promise<void> {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const response: any = await this.get('/api/user', {
```

Comment:
`any`? 😅 If you don't know the structure that is returned you can use `unkown` otherwise it is probably better to type it.